javascript/src/config.ts

```diff
@@ -146,8 +146,7 @@ export default () => {
       {
         name: 'HGT',
         label: t`Scaled Height`,
-        // Tmin_sp is multiplied by 10, so we divide by 10 here to get the real value
-        fn: 'math_e**(6.705 + (0.07443/10 * Tmin_sp))',
+        fn: 'math_e**(6.705 + (0.07443*Tmin_sp))',
```

Comment:
We're no longer multiplying temperatures by 10, so anywhere we divided by 10 has been deleted.